ModuleFifthTasks/Task1/LinqTask.cs

```diff
@@ -8,63 +9,77 @@ public static class LinqTask
     {
         public static IEnumerable<Customer> Linq1(IEnumerable<Customer> customers, decimal limit)
         {
-            throw new NotImplementedException();
+            return customers.Where(customer => customer.Orders.Sum(order => order.Total) > limit);
         }
 
         public static IEnumerable<(Customer customer, IEnumerable<Supplier> suppliers)> Linq2(
             IEnumerable<Customer> customers,
             IEnumerable<Supplier> suppliers
         )
         {
-            throw new NotImplementedException();
+            return customers.Select((customer, num) => new ValueTuple<Customer, IEnumerable<Supplier>>(customer, suppliers.Where(supplier => supplier.Country == customer.Country && supplier.City == customer.City)));
```

Comment:
Long lines are hard-readable.

```suggestion
            return customers.Select(customer =>
            (
                customer,
                suppliers.Where(supplier => supplier.Country == customer.Country
                     && supplier.City == customer.City)
            ));
```

Reply:
Fixed in d71bf5c.